web/packages/plugins/dmt-app/src/pages/SearchPage.tsx

```diff
@@ -450,6 +452,7 @@ export default ({ settings }: any) => {
   const [result, setResult] = useState<{ [key: string]: any }>({})
   const [queryError, setQueryError] = useState('')
   const [dataSources, setDataSources] = useState<DataSources>([])
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore